src/modules/pos/validator/components/ChangeCommission/Form/ChangeCommissionForm.test.js

```diff
@@ -158,22 +158,33 @@ describe('ChangeCommissionForm', () => {
       fireEvent.change(input, { target: { value } });
```

Comment:
NRTTL:
I cannot edit to make this 0.0024
<img width="205" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/c48c4b2f-bb8f-4e79-9df3-79b59361f9bb">